src/main/kotlin/com/example/noswimmingserver/domain/book/presentation/BookController.kt

```diff
@@ -0,0 +1,19 @@
+package com.example.noswimmingserver.domain.book.presentation
+
+import com.example.noswimmingserver.domain.book.presentation.dto.response.ReadBookResponse
+import com.example.noswimmingserver.domain.book.service.BookReadService
+import org.springframework.http.HttpStatus
+import org.springframework.web.bind.annotation.*
```

Comment:
🚫 **[ktlint]** <no-wildcard-imports> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wildcard import (cannot be auto-corrected)